queries.md

```diff
@@ -0,0 +1,158 @@
+1.
+Filter : {name :'Babelgum'}
+Project : {name:1, _id:0}
+Sort : 
+Collation : 
+Skip : 
+Limit :
+
+2.
+Filter : {number_of_employees:{$gt:5000}}
+Project : {name:1,_id:0}
+Sort : 
+Collation : 
+Skip : 
+Limit :20
+
+3.
+Filter : {$and:[{founded_year:{$gte:200}},{founded_year:{$lte:2005}}]}
```

Comment:
To make this shorter,  you can do this: `{founded_year: {$gte: 2000, $lte: 2005 }}`